ruby_connectors/lib/ruby_connectors/temporal/workflows/fetch_schema_workflow.rb

```
@@ -0,0 +1,13 @@
+
+module RubyConnectors
+  module Temporal
+    module Workflows
+      class FetchSchemaWorkflow < ::Temporal::Workflow
+        def execute(connector_id)
+          RubyConnectors::Temporal::Activities::FetchSchemaActivity.execute!(connector_id)
+        end
+      end
+    end
+  end
+end
```

rubocop

[convention] Final newline missing.

(Layout/TrailingEmptyLines)